src/main/java/com/mgmtp/radio/controller/v1/StationController.java

```diff
@@ -90,4 +92,11 @@ public StationController(StationService stationService) {
         return stationService.update(id, stationDTO);
     }
 
+    @PutMapping("/update-config/{id}")
```

Comment:
PUT method means update, so you should change url

Reply:
We means to update it @dat-ng